explainability-arrow/pom.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <parent>
+    <groupId>org.kie.trustyai</groupId>
+    <artifactId>trustyaiexplainability</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>explainability-arrow</artifactId>
+  <version>2.0.0-SNAPSHOT</version>
+  <name>TrustyAI Arrow</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.kie.trustyai</groupId>
+      <artifactId>explainability-core</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <!-- Arrow dependencies -->
+    <dependency>
+      <groupId>org.apache.arrow</groupId>
+      <artifactId>arrow-vector</artifactId>
+      <version>7.0.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.arrow</groupId>
+      <artifactId>arrow-compression</artifactId>
+      <version>7.0.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.arrow</groupId>
+      <artifactId>arrow-memory-netty</artifactId>
+      <version>7.0.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <version>5.8.2</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-buffer</artifactId>
+      <version>4.1.68.Final</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.flatbuffers</groupId>
+      <artifactId>flatbuffers-java</artifactId>
+      <version>2.0.3</version>
+    </dependency>
```

Comment:
Can you move these dependencies to [`dependencyManagement` section](https://github.com/trustyai-explainability/trustyai-explainability/blob/main/pom.xml#L44) (with [properties declaration](https://github.com/trustyai-explainability/trustyai-explainability/blob/main/pom.xml#L12) too)